tests/buildTools/src/test/scala/tests/ScipBuildToolSuite.scala

```diff
@@ -4,6 +4,71 @@ import com.sourcegraph.scip_java.{BuildInfo => V}
 
 class ScipBuildToolSuite extends BaseBuildToolSuite {
   override def tags = List(SkipWindows)
+
+  test("COURSIER_CREDENTIALS and COURSIER_REPOSITORIES are respected") {
+
+    val cli = sys.env.getOrElse("SCIP_JAVA_CLI", fail("wwaaaa"))
+
+    val Username = "hello"
+    val Password = "world"
+
+    val (requests, _) = PasswordProtectedServer(Username, Password).runWith {
+      run =>
+        val env = Map(
+          "COURSIER_REPOSITORIES" -> run.address.toString(),
+          "COURSIER_CREDENTIALS" -> s"localhost $Username:$Password"
+        )
+
+        val tmp = os.temp.dir(prefix = "scip-java")
```

Comment:
FYI, we use `FileLayout.fromString(..., root = workingDirectory)` in `BaseBuildToolSuite`, which makes it easier to create big directory structures without multiple `os.write(...)` invocations.

Reply:
I'll punt this to the next PR for sure 👍